### PR TITLE
modified http documentation 

### PR DESCRIPTION
**Pull request by nicokaegi**
# Pull Request Template

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: permissions.apib [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+### User_Permission
+ + user_permission_id: 63 (number, required)
+ + permission_id: 93 (number, required)
+ + user_id: 42 (number, required)
+
+# Group Permissions
+The Permissions
+
+## Permissions [/api/v1/permission]
+###get Permissionss [GET]
```
Comment (timothyhollabaugh) [inline permissions.apib]: Is this a typo?

Comment (nicokaegi) [reply permissions.apib]: are you referring to the double s in "get Permissionss"? then yes. ill fix it in a sec

Comment (timothyhollabaugh) [reply permissions.apib]: Yes

Comment (nicokaegi) [reply permissions.apib]: also could you tell which permissions I changed so I can change them back (i'm not sure how to view that here in the pull request page)

Comment (timothyhollabaugh) [reply permissions.apib]: If you go to the Files Changed tab, it shows the permissions changed for each file as
`100644 → 100755`, where 664 is the previous permissions and 755 are the new permissions

Comment (nicokaegi) [reply permissions.apib]: thanks for the tip :)

**Pull request merged**